core/bcast/bcast.go

```diff
@@ -146,39 +144,8 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty, pubkey core.
 		// Randao is an internal duty, not broadcasted to beacon chain
 		return nil
 	case core.DutyPrepareAggregator:
-		sub, ok := aggData.(core.SignedBeaconCommitteeSubscription)
-		if !ok {
-			return errors.New("invalid beacon committee sub")
-		}
-
-		_, err = b.eth2Cl.SubmitBeaconCommitteeSubscriptionsV2(ctx, []*eth2exp.BeaconCommitteeSubscription{&sub.BeaconCommitteeSubscription})
-		if err == nil {
-			return nil
-		}
-
-		// Ignore error as beacon node probably doesn't support v2 SubmitBeaconCommitteeSubscriptions
-		// endpoint (yet). Just try again with v1.
-
-		res, err := eth2exp.CalculateCommitteeSubscriptionResponse(ctx, b.eth2Cl, &sub.BeaconCommitteeSubscription, sub.CommitteeLength)
-		if err != nil {
-			return err
-		}
-
-		subs := []*eth2v1.BeaconCommitteeSubscription{{
-			ValidatorIndex:   res.ValidatorIndex,
-			Slot:             res.Slot,
-			CommitteeIndex:   res.CommitteeIndex,
-			CommitteesAtSlot: res.CommitteesAtSlot,
-			IsAggregator:     res.IsAggregator,
-		}}
-
-		err = b.eth2Cl.SubmitBeaconCommitteeSubscriptions(ctx, subs)
-		if err == nil {
-			log.Info(ctx, "Successfully submitted beacon committee subscription to beacon node",
-				z.Any("delay", b.delayFunc(duty.Slot)))
-		}
-
-		return err
+		// Beacon committee selections are only applicable to DVT, not broadcasted to beacon chain
```

Comment:
beacon node needs to support this endpoint in future? Or is it just for DVT?

Reply:
Beacon node could support it if they want by just returning the request. We could also support upstream DVTs if we want, but for now we do not.